agent/src/agent/scripts/reset_retry_counters.py

```diff
@@ -25,9 +25,13 @@ def main():
             continue
 
         try:
-            last_updated_in_min = int((datetime.now() - pipeline_.retries.last_updated).total_seconds() / 60)
-            if last_updated_in_min - constants.STREAMSETS_NOTIFY_RESET_AFTER_MIN > 0:
-                pipeline.manager.reset_pipeline_retries(pipeline_)
+            if not pipeline_.retries.last_updated:
```

Comment:
If `last_updated` is none then this record was created before we had `last_updated` column in the table.  So in this case I think more correct logic would be just to do `pipeline.manager.reset_pipeline_retries(pipeline_)` instead of setting it to now()